packages/web/src/services/AudiusBackend.ts

```diff
@@ -840,13 +914,22 @@ class AudiusBackend {
     } catch (err) {
       console.error(err)
     }
-    return feedItems.map((item) => {
-      if (item.playlist_id) return AudiusBackend.getCollectionImages(item)
+    return feedItems.map((item: Collection | Track) => {
+      if ((item as Collection).playlist_id) {
```

Comment:
i think you can do `if ('playlist_id' in item)` which then tells the rest of the branch item is Collection, and then also do we need to type (item: Collection | Track) or can that come from feedItem's type?

Reply:
Good call, yeah that all makes sense